Aplib.Core/Intent/Tactics/Tactic.cs

```diff
@@ -14,6 +14,11 @@ namespace Aplib.Core.Intent.Tactics
     public abstract class Tactic<TBeliefSet> : ITactic<TBeliefSet>
         where TBeliefSet : IBeliefSet
     {
+        /// <summary>
+        /// Gets or sets the guard of the tactic.
+        /// </summary>
+        protected System.Func<TBeliefSet, bool> _guard = _ => true;
```

Comment:
Why remove the autoproperty? It's not readonly or anything.

Reply:
It's not readonly because of a flaw in the code. I will suggest to open a backlog item for it, but it's caused by a problem that occurs at multiple places in the codebase.